deltachat-ffi/deltachat.h

```diff
@@ -343,6 +343,13 @@ char*           dc_get_blobdir               (const dc_context_t* context);
  *                    and watch the `DeltaChat` folder for updates (default),
  *                    0=do not move chat-messages
  *                    changes require restarting IO by calling dc_stop_io() and then dc_start_io().
+
+ * - `only_fetch_mvbox`
+ *                  = 1=Do not fetch messages from folders other than the
+ *                      `DeltaChat` folder.  Messages will still be fetched from the,
+ *                       spam folder and `sendbox_watch` will also still be respected
+ *                       if enabled.
+ *                    0=watch all folders normally (default)
```

Comment:
I also like your formatting better, but as the rest of the file is formatted differently, I'd personally format it like the rest (though i won't argue about this).